test/script/kabingurodo.js

```diff
@@ -0,0 +1,36 @@
+/* kabingurodo.js */
+
+ui.debug.addDebugData("kabingurodo", {
+	url: "6/6/g2g20920",
+	failcheck: [
+		[
+			"csAdjacent",
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. . . . . . /. . . # . . /# . . . . . /. . . # . . /. . . # . . /. . # . . # /"
+		],
+		[
+			"cuDivideRB",
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. . . . . . /# . . # . . /. # . . . . /. . # . . . /. . . # . . /. . # . . . /"
+		],
+		[
+			"kabinNoTurns",
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. . # . . . /# . . . . . /. . # . # . /. # . . . # /# . . # . . /. . # . . # /"
+		],
+		[
+			"kabinNoTurns",
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. . # . . . /# . . # . # /. . . . . . /. # . . # . /# . . # . . /. . # . . # /"
+		],
+		[
+			"kabinOneTurn",
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. . # . . . /# . . # . # /. # . . . . /. . # . # . /# . . # . . /. . . . . # /"
+		],
+		[
+			"kabinOneTurn",
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. . # . . # /# . . # . . /. . . . # . /. . # . . . /# . . # . . /. . # . . # /"
+		],
+		[
+			null,
+			"pzprv3/kabingurodo/6/6/1 . . . . . /. . 1 . 1 . /. . . . . . /1 . . . . . /. . 1 . . 1 /. . . 1 . . /. + # + + + /# + . # . # /+ + + + + + /. + # + # + /# + . # + . /+ + # . + # /"
+		]
+	],
+	inputs: []
```

Comment:
The functions in `MouseEvent` are not covered by tests. Could you add a few input tests for the Editing part here?